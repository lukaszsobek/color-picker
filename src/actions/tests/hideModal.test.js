import { hideModal } from "../";

it("Hides modal",() => {
    expect(hideModal()).toMatchSnapshot();
});