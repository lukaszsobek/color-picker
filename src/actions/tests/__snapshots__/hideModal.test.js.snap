// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hides modal 1`] = `
Object {
  "isModalVisible": false,
  "type": "HIDE_MODAL",
}
`;
